portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,5 @@
+function getRandomQuote(){
+    fetch('/data').then(response => response.text()).then((quote) => {
+        document.getElementById('quote-container').innerText = quote;
+    })
```

Comment:
While not necessary, I'd suggest structuring your promises as such to help make it more readable. If you were to have a long chain it may get confusing trying to read it all on one line
```suggestion
    fetch('/data')
      .then(response => response.text())
      .then((quote) => {
        document.getElementById('quote-container').innerText = quote;
      })
```